packages/iov-faucets/src/iovfaucet.ts

```diff
@@ -0,0 +1,25 @@
+import axios from "axios";
+
+import { Address, TokenTicker } from "@iov/bcp-types";
+
+export class IovFaucet {
+  private readonly baseUrl: string;
+
+  constructor(baseUrl: string) {
+    if (!baseUrl.match(/^https?:\/\//)) {
+      throw new Error("Expected base url to start with http:// or https://");
+    }
+
+    // Strip trailing /
+    const strippedBaseUrl = baseUrl.replace(/(\/)+$/, "");
+    this.baseUrl = strippedBaseUrl;
+  }
+
+  public async credit(address: Address, ticker: TokenTicker): Promise<void> {
+    const body = {
+      address: address,
+      ticker: ticker,
+    };
+    await axios.post(this.baseUrl + "/credit", body);
```

Comment:
In bovfaucet we have an url (with /faucet) and in iovfaucet we hace url + api (/credit), can we standardize this posts?

Reply:
No response/return from post?

Reply:
> can we standardize this posts?

`baseUrl` is the new default, allowing the class to add other functions later (like available tokens from /status).

But since `BovFaucet` is already deprecated, I don't want to do any incompatible change there anymore.

> No response/return from post?

Right, just `void` or an error. No message is good message ;)